src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/tasklets/PopulationStatisticsGeneratorStep.java

```diff
@@ -75,4 +82,66 @@ public RepeatStatus execute(StepContribution contribution, ChunkContext chunkCon
 
         return RepeatStatus.FINISHED;
     }
+
+    public static URI getStatsBaseUri(String outputDirStatistics, String studyId, String fileId) throws URISyntaxException {
```

Comment:
Ok, I will do it for statistics, and in the PR of annotation jobParameters, I will remove VepUtils.